modules/ossm-verifying-multi-cluster-topology.adoc

```diff
@@ -118,7 +118,7 @@ $ oc --context="${CTX_CLUSTER2}" apply \
 [source,terminal]
 ----
 $ oc --context="${CTX_CLUSTER2}" apply \
-  -f https://raw.githubusercontent.com/istio/istio/${ISTIO_VERSION}/samples/helloworld/helloworld.yaml \
+  -f https://raw.githubusercontent.com/openshift-service-mesh/istio/${ISTIO_VERSION}/samples/helloworld/helloworld.yaml \
```

Comment:
I noticed this one was left with the `{ISTIO_VERSION}`, but it looks like others were changes to release-1.24... is that correct?